test/source/tests/page_recipe.ts

```diff
@@ -69,7 +69,8 @@ export class SetupPageRecipe extends PageRecipe {
     } else if (backup === 'email') {
       throw new Error('tests.setup_manual_create options.backup=email not implemented');
     } else if (backup === 'file') {
-      throw new Error('tests.setup_manual_create options.backup=file not implemented');
+      await settingsPage.waitAny('@action-backup-download'); // Only check if there is a checkbox.
+      return;
```

Comment:
This doesn't actually proceed with the backup - the test is not useful.

We should test scenarios that user would do - set up the extension (completely) using this backup method.